main.nf

```diff
@@ -62,3 +62,16 @@ workflow {
 workflow.onComplete {
 	log.info ( workflow.success ? "\nDone! Open the following report in your browser --> $params.outdir/multiqc_report.html\n" : "Oops .. something went wrong" )
 }
+
+output {
+  directory params.outdir
+  mode 'copy'
+
+  'fastqc' {
+    path '.'
+  }
+
+  'multiqc' {
+    path '.'
+  }
```

Comment:
Because MULTIQC is called in the main workflow and the current prototype allows `publish:` in anonymous workflows, this rule could also be in the main workflow:
```groovy
workflow {
  // ...
  publish:
  MULTIQC.out >> '.'
}
```

Reply:
Neat